.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
please ignore it

Reply:
[editorconfig](https://editorconfig.org/) is useful as a standard that tells code editors how to behave (it works with every code editor). I understand that the linter could fix style errors later, but if my editor mistakes the code indenting for 4 spaces instead of 2 for example, it makes it harder to edit the code to begin with.

I'm fine to add it to .gitignore if you don't like it, but I do think it could be useful to other devs.

Reply:
ok, keep it